### PR TITLE
Replace Travis with Github Actions 

### PR DESCRIPTION
**Pull request by domjacks**
📺 What

Move from Travis to GitHub Actions after Travis reduced support for non-paid plans:
https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

This also sets our Node version to 14.

🛠 How

Created two new workflows in `/.github/workflows/`:
* Pull Request Checks: Runs when a new PR is raised into the Master branch. Runs Tests
* NPM Publish: Runs whenever a new tag is created. Creates build and publishes the package to NPM.

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - '*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
```
Comment (matt-stephenson) [inline .github/workflows/npm-publish.yml:15]: Possibly bump to LTS

File: README.md [hunk 2]
```diff
@@ -238,7 +238,9 @@ See [here](https://github.com/bbc/bigscreen-player/wiki/Mocking-Bigscreen-Player
 3. Get a review from the core team.
 4. If the PR checks are green. The core team can merge to master.
 5. Automation takes care of the package versioning.
-6. Publishing to NPM is handled with our [Travis CI integration](https://github.com/bbc/bigscreen-player/blob/master/.travis.yml).
+6. Publishing to NPM is handled with our [GitHub Actions CI integration](https://github.com/bbc/bigscreen-player/blob/master/.github/workflows/deploy-requests.yml).
```
Comment (matt-stephenson) [inline README.md:241]: Is this yml filename right?

**Review by matt-stephenson — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (domjacks) [reply README.md:241]: Ah good catch, I renamed!

File: .github/workflows/npm-publish.yml (hunk 1 above)
Comment (domjacks) [reply .github/workflows/npm-publish.yml:15]: I've switched to 14. Will see if there's any impact

**Review by matt-stephenson — APPROVED** [review]

**Review by ShiningTrapez — APPROVED** [review]

**Pull request merged**